server/src/handler_monitor.c

```diff
@@ -53,84 +53,79 @@ onion_connection_status handler_get_trains(void *_, onion_request *req,
 			                       train_grabbed(query.ids[i]) ? "yes" : "no");
 		}
 		bidib_free_id_list_query(query);
-		char response[trains->len + 1];
-		strcpy(response, trains->str);
+		onion_response_printf(res, "%s", trains->str);
+		syslog_server(LOG_INFO, "Request: Get available trains - finished");
 		g_string_free(trains, true);
-		onion_response_printf(res, "%s", response);
-		syslog_server(LOG_NOTICE, "Request: Get available trains");
 		return OCS_PROCESSED;
 	} else {
-		syslog_server(LOG_ERR, "Request: Get available trains - system not running or "
-		              "wrong request type");
+		syslog_server(LOG_ERR, 
```

Comment:
Since this is a getter, shouldn't the message be at INFO level?

Reply:
No, in an error/failure case it should be LOG_ERR IMO.